domains/amila.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Y-AmilaNirmal",
+        "email": "yamilanirmal@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "https://y-amilanirmal.github.io/y-amilanirmal"
```

Comment:
Invalid CNAME. CNAME should not contain slashes, protocols, or ://.

Reply:
yesterday  I  did change my user name Y-AmilaNirmal to y-amila-nirmal so is if you can changed my json details to that

{
    "owner": {
        "username": "y-amila-nirmal",
        "email": "yamilanirmal@gmail.com"
    },

    "record": {
        "CNAME": "https://y-amila-nirmal.github.io/y-amila-nirmal"
    }
}

I am grateful to you. Consider this a little. 🙂

Reply:
@y-amila-nirmal You will need to open a new PR updating your file.

Reply:
"You will need to open a new PR updating your file." <- I didn't that, but its working now thank you for help